app-consumer/cmd/consumer/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+
+	"activity_tracker/app-consumer/internal/kafka"
+	"activity_tracker/app-consumer/internal/service"
+	"activity_tracker/app-consumer/internal/storage"
+	"activity_tracker/config"
+)
+
+func main() {
+	log.Println("Запуск App Consumer...")
+
+	pgDSN := config.LoadDSN()
```

Comment:
так будет информативней, что-за dsn
```suggestion
	pgDSN := config.LoadPGDSN()
```